modules/react/common/lib/AccessibleHide.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {CSSProperties} from '@workday/canvas-kit-react/tokens';
+import {createStyles, CSProps, handleCsProp} from '@workday/canvas-kit-styling';
+import {createComponent} from './utils/components';
+
+/**
+ * A utility to visually hide content, while still making accessible to screen readers
+ * See https://a11y-101.com/development/skip-link
+ */
+export const accessibleHide: CSSProperties = {
+  clip: 'rect(1px, 1px, 1px, 1px)', // Deprecated but still used by most browsers, clip-path will be taking its place soon.
+  clipPath: 'polygon(0px 0px, 0px 0px, 0px 0px, 0px 0px)',
+  position: 'absolute',
+  overflow: 'hidden',
+  whiteSpace: 'nowrap',
+  height: '1px',
+  width: '1px',
+  margin: '-1px',
+  padding: 0,
+  border: 0,
+};
```

Comment:
Do you need to put this in a seperate object for clip and clip path properties instead of just in a `createStyles` object?

Reply:
This already existed as an export – I don't want to create a breaking change.
https://github.com/Workday/canvas-kit/blob/master/modules/react/common/lib/styles/accessibleHide.ts#L7-L18